app/person/admin.py

```diff
@@ -1,5 +1,18 @@
-from django.contrib import admin
-
+from django.contrib import admin, messages
+from rollup.utils import load_rollup
+from django_object_actions import DjangoObjectActions
 from .models import Person
-# Register your models here.
-admin.site.register(Person)
+
+# Adds a button to the Person Admin page to allow users to manually update the
+# directory screens with the most up-to-date data
+class PersonAdmin(DjangoObjectActions, admin.ModelAdmin):
+
+  def UpdateScreens(self, request, queryset):
+    load_rollup()
+    self.message_user(request, 'Successfully updated screens with revised person and place data.', level=messages.SUCCESS, fail_silently=False)
+  
+  UpdateScreens.label = 'Update screens'
```

Comment:
Can we add an `Update.Screen.short_description` to explain what the button will actually do (e.g. "Manually refresh the screens with the most recent data")? 

I'm not sure if there's an easy way to do this, but it could also be good to have that appear as a persistent message on the page as well, since "Update Screens" may not be totally clear to most users.

Reply:
@jonseitz Thanks Jonathan! I added the `short_description` in 55ad23ebd417, but I haven't found a way to add a persistent message on load of the admin page. It seems from the docs that to add a message, it would have to be in response to some action like clicking a button. Perhaps I can change the text of the button to something more descriptive - originally Chris had suggested "Push to Screens" as button text, but I thought that the word "push" could be too technical of a term?

Reply:
OK. I don't think a persistent message is totally necessary, and "Update Screens" is still probably fine as a button label -- ultimately I think it's going to be a question of user training more than anything we do in code.